src/ml_performance_monitoring/monitor.py

```diff
@@ -65,7 +65,6 @@ def __init__(
         send_data_metrics=False,
         features_columns: Optional[List[str]] = None,
         labels_columns: Optional[List[str]] = None,
-        label_type: Optional[str] = None,
```

Comment:
I thinks that we don't need to remove it in the next few week so we won't break the code to the users

Reply:
done